css/index.css

```diff
@@ -143,4 +153,104 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+    
+.header nav{
+    width:80vw;
+    display:flex;
+    justify-content: space-evenly;
+    align-items:center;
+}
+header{
+    width: 60%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px 2%;
+    font-size: 14px;
+
+}
+
+.container .image-container{
+    display:flex;
+    justify-content: center;
+
+}
+
+a{
+    text-decoration: none;
+    font-size: 16px;
+    color:black;
+
+}
+
+a:hover{
+    color: #db3a45;
+    transition: 0.3s;
+}
+
```

Comment:
Nice job allowing the .box class to style everything needed for each individual box and overriding only the color!